src/_P118_Itho.ino

```diff
@@ -0,0 +1,548 @@
+//#######################################################################################################
+//############################## Plugin 118: Itho ventilation unit 868Mhz remote ########################
+//#######################################################################################################
+
+// author :jodur, 	13-1-2018
+// changed :jeroen, 2-11-2019
+// changed :svollebregt, 30-1-2020 - changes to improve stability: volatile decleration of state,
+//          disable logging within interrupts unles enabled, removed some unused code,
+//          reduce noInterrupts() blockage of code fragments to prevent crashes
+//			svollebregt, 16-2-2020 - ISR now sets flag which is checked by 50 per seconds plugin call as
+//			receive ISR with Ticker was the cause of instability. Inspired by: https://github.com/arnemauer/Ducobox-ESPEasy-Plugin
+//			svollebregt, 11-04-2020 - Minor changes to make code compatible with latest mega 20200410, removed SYNC1 option for now;
+//			better to change this value in the Itho-lib code and compile it yourself
+//			svollebreggt, 13-2-2021 - Now uses rewirtten library made by arjenhiemstra: https://github.com/arjenhiemstra/IthoEcoFanRFT
+
+// Recommended to disable RF receive logging to minimize code execution within interrupts
+
+// List of commands:
+// 1111 to join ESP8266 with Itho ventilation unit
+// 9999 to leaveESP8266 with Itho ventilation unit
+// 0 to set Itho ventilation unit to standby
+// 1 - set Itho ventilation unit to low speed
+// 2 - set Itho ventilation unit to medium speed
+// 3 - set Itho ventilation unit to high speed
+// 4 - set Itho ventilation unit to full speed
+// 13 - set itho to high speed with hardware timer (10 min)
+// 23 - set itho to high speed with hardware timer (20 min)
+// 33 - set itho to high speed with hardware timer (30 min)
+
+//List of States:
+
+// 1 - Itho ventilation unit to lowest speed
+// 2 - Itho ventilation unit to medium speed
+// 3 - Itho ventilation unit to high speed
+// 4 - Itho ventilation unit to full speed
+// 13 -Itho to high speed with hardware timer (10 min)
+// 23 -Itho to high speed with hardware timer (20 min)
+// 33 -Itho to high speed with hardware timer (30 min)
+
+// Usage for http (not case sensitive):
+// http://ip/control?cmd=STATE,1111
+// http://ip/control?cmd=STATE,1
+// http://ip/control?cmd=STATE,2
+// http://ip/control?cmd=STATE,3
+
+// usage for example mosquito MQTT
+// mosquitto_pub -t /Fan/cmd -m 'state 1111'
+// mosquitto_pub -t /Fan/cmd -m 'state 1'
+// mosquitto_pub -t /Fan/cmd -m 'state 2'
+// mosquitto_pub -t /Fan/cmd -m 'state 3'
+
+
+// This code needs the library made by 'supersjimmie': https://github.com/supersjimmie/IthoEcoFanRFT/tree/master/Master/Itho
+// A CC1101 868Mhz transmitter is needed
+// See https://gathering.tweakers.net/forum/list_messages/1690945 for more information
+// code/idea was inspired by first release of code from 'Thinkpad'
+
+#include <SPI.h>
+#include "IthoCC1101.h"
```

Comment:
Moving the library includes after the `#ifdef USES_P118` will eliminate the need to include SPI.h, and make the compiler job a tad lighter for the builds that don't have this plugin active.